### PR TITLE
Add possibility to run tests maximized

### PR DESCRIPTION
**Pull request by ScrewTSW**
### What does this PR do?
This PR adds option to run the tests in a maximized window to prevent cropping by `ffmpeg`

### Screenshot/screencast of this PR
The driver helper just maximizes the Chrome window after Login

### What issues does this PR fix or reference?
#18482 

### How to test this PR?
* Change any mocha file to use LoginMaximized.spec instead of Login.spec
* Run the tests and verify that the window is maximized after Login

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by ScrewTSW** [comment]
[ci-build]

**Review by Katka92 — COMMENTED** [review]
What about having an extra parameter instead of a completely separated new test? I mean something like `TS_SELENIUM_MAXIMIZE_SCREEN` by default to false and then just have the if in current Login.spec.ts class. It will be easier to enable/disable the full screen without a need to rebuild the whole image.

**Comment by ScrewTSW** [comment]
> What about having an extra parameter instead of a completely separated new test? I mean something like `TS_SELENIUM_MAXIMIZE_SCREEN` by default to false and then just have the if in current Login.spec.ts class. It will be easier to enable/disable the full screen without a need to rebuild the whole image.

That's a really good idea... I feel dumb for not thinking about it, thanks for the suggestion!

File: tests/e2e/tests/login/Login.spec.ts [hunk 1]
```diff
@@ -19,4 +19,9 @@ suite('Login test', async () => {
         await driverHelper.navigateToUrl(TestConstants.TS_SELENIUM_BASE_URL);
         await loginPage.login();
     });
+    if (TestConstants.TS_SELENIUM_LAUNCH_FULLSCREEN){
+        test('Maximize windows', async() => {
```
Comment (Katka92) [inline tests/e2e/tests/login/Login.spec.ts]: I don't think that maximizing windows is something we want to test - I would include it in the Login test itself.

Comment (ScrewTSW) [reply tests/e2e/tests/login/Login.spec.ts]: Fixed

**Comment by ScrewTSW** [comment]
[ci-build]

**Comment by Katka92** [comment]
[ci-build]

**Review by Katka92 — APPROVED** [review]

**Pull request merged**